tools/ovc/openvino/tools/ovc/cli_parser.py

```diff
@@ -90,6 +89,27 @@ def single_input_to_input_cut_info(input: [str, tuple, list, PartialShape, Type,
 
     raise Exception("Unexpected object provided for input. Expected tuple, Shape, PartialShape, Type or str. Got {}".format(type(input)))
 
+def is_single_input(input: [str, tuple, list]):
```

Comment:
Type annotation is a bit confusing, this is not a list of specified types.

Reply:
Corrected type annotation and added comment to the method.